config_test.go

```diff
@@ -27,3 +27,20 @@ packages:
 
 	assert.Equal(t, pkg, Package{Repo: "github.com/grpc/grpc-go"})
 }
+
+func TestNotAlphabetical(t *testing.T) {
+	path, clean := TempFile(t, `
+
+url: google.golang.org
+packages:
+  grpc:
+    repo: github.com/grpc/grpc-go
+  atomic:
+    repo: github.com/uber-go/atomic
+
+`)
+	defer clean()
+
+	_, err := Parse(path)
+	assert.Error(t, err, "YAML configuration is not listed alphabetically")
```

Comment:
It might be worth verifying the error message.

```go
if assert.Error(t, err, "expected Parse to fail") {
    assert.Contains(t, err.Error(), "must be alphabetically ordered",
        "error message did not match")
}
```